CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- `list-context.media-list` block that provides a list context of images and videos
+  
+### Changed
+- Improved other apps' CSS Handles imports
```

Comment:
```suggestion
- Improved other apps' CSS Handles imports.
```

Reply:
This entry is a bit generic, could you name the apps?

Reply:
Sure!